### ISSUE TITLE
--primary-only still grabbing Additional Sets

### ISSUE THREAD

**Issue by Furglitch** [CollinHeist/TPDbCollectionMaker#6]
I'm having an issue where using `--primary-only` or `-p` will still give me poster images for Additional Sets

Example using Sevi's _Marvel Cinematic Universe_ poster set - https://theposterdb.com/set/120593

* Extracted the Inner HTML and put it into MCU.html in /pages
* Execute the python script via powershell
* The outputted YML includes shows and movies from Additional Sets, like _Daredevil_ from '_Additional Set - Marvel Television_' or _Blade_ from '_Additional Set - In Association With Marvel_'

To note, I am using a PowerShell script just to batch execute and put things in folders with the .yml filenames being the set name. I've included that below.
```powershell
# get list of HTML files within /pages
$htmlFiles = Get-ChildItem -Path ./pages -Filter *.html

# For loop for every file in the /pages folder
foreach ($htmlFile in $htmlFiles) {
    
    # get the set name from the html
    $htmlContent = Get-Content -Path $htmlFile.FullName -Raw
    if ($htmlContent -match '<title>\s*(.*?)\s+Set\s+By\s+(.*?)\s*\|\s+The\s+Poster\s+Database\s+\(TPDb\)\s*</title>') { $setName = $matches[1] }

    # execute main.py with -p and -q for the html, outputting to /output/SET-NAME.yml
    poetry run python main.py --primary-only --always-quote "./pages/$htmlFile" > "./output/$SetName.yml"
}
```
But this still happens when I just use `poetry run python main.py --primary-only --always-quote "./pages/MCU.html" > "./output/MCU.yml"`
Also happens regardless if I use `-p` or `--primary-only` and/or remove `--always-quote` or `-p`
Also happens if I don't use folders, and just have the html and yml output in /TPDbCollectionMaker

I can give the HTML if necessary. Just didn't want to put a whole wall of text from that.

**Comment by Furglitch** [comment]
It seems like the problem is that the script doesn't switch to using the filtered class that is parsed by `if args.primary_only:` when grabbing posters. 

I've made a pull request that fixed this on my end (#7), but I'm not the most knowledgeable in Python, so it might need a lookover.

**Comment by CollinHeist** [comment]
Thanks for identifying the fix! Looks good.
